packages/ui/src/components/vuestic-components/va-modal/VaModal.vue

```diff
@@ -92,6 +94,7 @@ import VaIcon from '../va-icon'
 
 class ModalProps {
   modelValue = prop<boolean>({ type: Boolean, default: false })
+  teleportTarget = prop<string>({ type: String, default: '#app' })
```

Comment:
I don't think users will understand the meaning of the `teleportTarget` prop. Need a better name.